### PR TITLE
SWS-955

### PR DESCRIPTION
**Pull request by jaminh**
Add method for configuring SAML callback.

### CODE REVIEW

File: spring-ws-security/src/test/java/org/springframework/ws/soap/security/wss4j2/AxiomWss4jMessageInterceptorSamlTest.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package org.springframework.ws.soap.security.wss4j2;
+
+public class AxiomWss4jMessageInterceptorSamlTest extends Wss4jMessageInterceptorSamlTestCase {
```
Comment (gregturn) [inline spring-ws-security/src/test/java/org/springframework/ws/soap/security/wss4j2/AxiomWss4jMessageInterceptorSamlTest.java]: What does this test class do? I realize it extends an abstract test case. But it looks like nothing is configured here specifically based on Axiom.

File: spring-ws-security/src/test/java/org/springframework/ws/soap/security/wss4j2/SaajWss4jMessageInterceptorSamlTest.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package org.springframework.ws.soap.security.wss4j2;
+
+public class SaajWss4jMessageInterceptorSamlTest extends Wss4jMessageInterceptorSamlTestCase {
+
```
Comment (gregturn) [inline spring-ws-security/src/test/java/org/springframework/ws/soap/security/wss4j2/SaajWss4jMessageInterceptorSamlTest.java]: Same question for this test class regarding Saaj.

**Comment by jaminh** [comment]
In the Wss4jTestCase class, which most if not all of the tests classes extend, looks at the beginning of the class name and will generate the SOAP message used in the test with the appropriate message factory (Axiom or Saaj) based on that. This is how most of the Wss4j tests were working so I tried following that convention.

**Comment by gregturn** [comment]
Thanks @jaminh. Now I can see how it's operating.

**Comment by gregturn** [comment]
Resolved via https://github.com/spring-projects/spring-ws/commit/e34fb937d155f6c0542dd7aca359c4449b6ce51a

**Pull request closed without merging**